backend/api/admin/src/main/java/co/airy/core/api/admin/ChannelsController.java

```diff
@@ -86,7 +86,7 @@ public ChannelsController(Stores stores) {
         final String sourceChannelId = requestPayload.getName();
         final String sourceIdentifier = "chatplugin";
 
-        final String channelId = UUIDv5.fromNamespaceAndName(sourceIdentifier, sourceChannelId).toString();
```

Comment:
The UUID 5 makes the behavior worse here: Every user that calls their channel `X` would get the same channel id here, because the source identifier is constant